helper/lora-packet-tracker.h

```diff
@@ -66,7 +66,7 @@ struct RetransmissionStatus
 };
 
 typedef std::map<Ptr<Packet const>, MacPacketStatus> MacPacketData;
-typedef std::map<Ptr<Packet const>, PacketStatus> PhyPacketData;
+typedef std::multimap<Ptr<Packet const>, PacketStatus> PhyPacketData;
```

Comment:
Using a multimap here seems to work fine, however I would also consider the possibility of having just one entry per (possibly re-transmitted) packet, and tracking re-transmissions inside the `PacketStatus`. I might need to take some more time to look at this and figure out what the cleanest option is.

Reply:
If you can think of a cleaner/better way that would be good.